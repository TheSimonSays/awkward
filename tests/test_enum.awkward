enum Color {
    Red,
    Green,
    Blue
}

let color = Color.Red;
print(color);

if (color == Color.Red) {
    print("It's red!");
}

enum Status {
    Pending,
    Success,
    Error
}

let status = Status.Success;
print(typeof(status));
