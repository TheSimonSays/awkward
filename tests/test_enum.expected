Color.Red
It's red!
enum
